libraries/shared/watcher/storage_watcher.go

```diff
@@ -266,7 +265,6 @@ func (watcher StorageWatcher) handleTransformError(transformErr error, diff type
 		} else if isCommonTransformError(transformErr) {
 			logrus.Tracef("error transforming diff: %s", transformErr.Error())
 		} else {
-			logrus.Infof("error transforming diff: %s", transformErr.Error())
 			return transformErr
```

Comment:
These two errors are bubbling up and eventually are logged [here](https://github.com/makerdao/vulcanizedb/blob/prod/cmd/execute.go#L161) in the top level `execute` command. 

The piece I'm not sure about is how to downgrade some specific errors that are bubbled to `execute` so that they won't be sent to sentry. For example:
```json
{"SubCommand":"execute","level":"fatal","msg":"error executing storage watcher: error transforming diff: error executing storage transformer: error inserting vat debt 1446022561362437719942102316255726955140806884038754047 from diff ID 7890761: pq: insert or update on table \"vat_debt\" violates foreign key constraint \"vat_debt_header_id_fkey\"","time":"2021-01-23T00:38:17Z"}

```
This error is likely due to the fact that we're trying to insert a `maker.vat_debt` record and associate with a header that no longer exists in the due to a reorg. Since this is an error directly from postgres, I don't think we're able to match on it like we're doing for other common/expected errors (e.g. `errors.Is(err, sql.ErrNoRows)`). So we could match on it based on comparing the error string, but I'm not sure how future proof that is.